config/default.yaml

```diff
@@ -88,3 +88,4 @@ guidanceNavigation:
       url: /guidance/publish-data-on-your-website
     - label: Keep your data up to date
       url: /guidance/keep-your-data-up-to-date
+tablePageLength: 50
```

Comment:
Do you need to update the schema on `config/utils.js`?

Reply:
weird, I didn't get any valibot error??
I'll add it in now anyways

Reply:
yeah I raised this with @rosado a few times, I think we should make it throw an error